### PR TITLE
Make ban action more reflective of the attack type

### PR DESCRIPTION
**Pull request by DigiDaz**

### CODE REVIEW

File: debian/resources/fail2ban/jail.local [hunk 1]
```diff
@@ -1,33 +1,67 @@
-[freeswitch]
+[freeswitch-udp]
 enabled  = true
 port     = 5060,5061,5080,5081
 protocol = all
 filter   = freeswitch
 logpath  = /var/log/freeswitch/freeswitch.log
-action   = iptables-allports[name=freeswitch, protocol=all]
```
Comment (moteus) [inline debian/resources/fail2ban/jail.local]: Is it possible block tcp and udp. I see no reason why not block tcp as whell

**Comment by DigiDaz** [comment]
Moteus,

This is exactly what we have done in the commit.

Unfortunately from reading the fail2ban forums, we cannot have more than one protocol per action.

We should be able to by passing protocol=all or protocol=0 but both of these fail.

Instead I have created separate blocks, one for TCP, one for UDP.

**Comment by moteus** [comment]
I mean that if we detect intrusion on udp we should close tcp as well. May be it possible add second action for tcp to udp jail and udp to tcp one.

**Comment by DigiDaz** [comment]
Moteus, I may be wron but I think this will already be the case as they are using the same filter

**Pull request merged**